en/ui-system/components/editor/videoplayer.md

```diff
@@ -17,51 +17,51 @@ For more information about VideoPlayer's scripting interface, please refer to th
+| **component**       | Script component name.         |
+| **handler**         | Specify a callback, when the video player is about to playing or paused, it will be called. There is a parameter in the callback which indicate the state of played videos.|
+| **customEventData** | The user specifies that any string is passed in as the last parameter of the event callback |
 
 For more information, please refer to the [Component.EventHandler Class](../../../api/en/classes/event.eventhandler.html) documentation.
 
 ### Parameter of VideoPlayerEvent
 
 | Name           | Function Explanation          |
-| -------------- | -----------                   |
-| NONE           | None                          |
-| PLAYING        | Video is playing.             |
-| PAUSED         | Video is paused.              |
-| STOPPED        | Video is stopped.             |
-| COMPLETED      | Video is completed.           |
-| META_LOADED    | Video's meta data is loaded.  |
-| READY_TO_PLAY  | Video is ready to play.       |
-| ERROR          | Video Trigger Error           |
-| CLICKED        | Video is clicked by the user. (Only supports Web platform.) |
+| :-------------- | :-----------                   |
+| **NONE**           | None                          |
+| **PLAYING**        | Video is playing.             |
+| **PAUSED**         | Video is paused.              |
+| **STOPPED**        | Video is stopped.             |
+| **COMPLETED**      | Video is completed.           |
+| **META_LOADED**    | Video's meta data is loaded.  |
+| **READY_TO_PLAY**  | Video is ready to play.       |
+| **ERROR**          | Video Trigger Error           |
+| **CLICKED**        | Video is clicked by the user. (Only supports Web platform.) |
 
-**Note:** On iOS platform, due to the platform limitations, the CLICKED event can't be fired when **VideoPlayer** is in fullscreen mode. If you want to let the Video played in fullscreen and also fire the CLICKED event properly, you should use a Widget component to hack the **VideoPlayer's** size.
+> **Note**: On iOS platform, due to the platform limitations, the CLICKED event can't be fired when **VideoPlayer** is in fullscreen mode. If you want to let the Video played in fullscreen and also fire the CLICKED event properly, you should use a Widget component to hack the **VideoPlayer's** size.
 
-For more information, please refer to the [VideoPlayer Events](../../../api/en/classes/video.videoplayer.html#videoplayerevent) or [21.video-player](https://github.com/cocos-creator/test-cases-3d/tree/v3.0/assets/cases/ui/21.video-player) of the test-cases-3d samples bundled with Creator.
+For more information, please refer to the [VideoPlayer Events](../../../api/en/classes/VideoPlayer.html#events) documentation or the [21.video-player example](https://github.com/cocos-creator/test-cases-3d/tree/v3.0/assets/cases/ui/21.video-player) in the test-cases-3d samples bundled with __Cocos Creator__.
```

Comment:
The original link is correct and does not need to be modified.

Reply:
I didn't change this, it happened when I did a `git pull` from the `v3.0` branch. See, they are incorrect in that branch, every link you mention: https://raw.githubusercontent.com/cocos-creator/creator-docs/v3.0/en/ui-system/components/editor/videoplayer.md

I fixed them now in `v3.0-release`, i.e. this PR.